### ISSUE TITLE
millis should be orthogonal in DERGeneralizedTime constructor

### ISSUE THREAD

**Issue by rapropos** [kjur/jsrsasign#138]
I believe there is a potentially unintended inability to set the "millis" property when creating a DERGeneralizedTime if other properties are also specified. It would make more sense to me to separate it into its own conditional, instead of how it is chained now.

```
if (typeof params != "undefined") {
        if (typeof params['str'] != "undefined") {
            this.setString(params['str']);
        } else if (typeof params == "string" && params.match(/^[0-9]{14}Z$/)) {
            this.setString(params);
        } else if (typeof params['hex'] != "undefined") {
            this.setStringHex(params['hex']);
        } else if (typeof params['date'] != "undefined") {
            this.setByDate(params['date']);
        } else if (params.millis === true) {
            this.withMillis = true;
        }
    }
```

could become

```
if (typeof params != "undefined") {
        if (typeof params['str'] != "undefined") {
            this.setString(params['str']);
        } else if (typeof params == "string" && params.match(/^[0-9]{14}Z$/)) {
            this.setString(params);
        } else if (typeof params['hex'] != "undefined") {
            this.setStringHex(params['hex']);
        } else if (typeof params['date'] != "undefined") {
            this.setByDate(params['date']);
        } 
        if (params.millis === true) {
            this.withMillis = true;
        }
    }
```

Please feel free to ignore this if the current exclusive behavior is intended.

**Comment by kjur** [comment]
Thank you for your advice. I'll check it later.

**Comment by kjur** [comment]
Fixed in 5.0.5 released today. Thanks
https://github.com/kjur/jsrsasign/releases/tag/5.0.5
